compiler/rustc_parse/locales/en-US.ftl

```diff
@@ -710,9 +710,12 @@ parse_zero_chars = empty character literal
 parse_lone_slash = invalid trailing slash in literal
     .label = {parse_lone_slash}
 
-parse_unskipped_whitespace = non-ASCII whitespace symbol '{$ch}' is not skipped
+parse_unskipped_whitespace = whitespace symbol '{$ch}' is not skipped
```

Comment:
I meant none of the changes except this line (and tests) are necessary, `UnskippedAsciiWhitespaceWarning` and `UnskippedAsciiWhitespace` can be removed.

Reply:
sorry for the misinterpretation...